packages/analyticsdx-template-lint/test/unit/schemas/testfiles/readiness/valid/all-fields.json

```diff
@@ -98,7 +98,7 @@
         {
           "field": "NumberOfEmployees",
           "operator": "NotEqual",
-          "value": "1"
+          "value": null
```

Comment:
Nit: There's a `readiness/valid/nulls.json` that has the other allowed nulls in it.